include/bluetooth/mesh/time_srv.h

```diff
@@ -46,19 +46,46 @@ struct tm;
  * @brief Time Server model composition data entry.
  *
  * @param[in] _srv Pointer to a @ref bt_mesh_time_srv instance.
+ * @param ...	   Optional Time Server metadata if application is
+ *		   compiled with Large Composition Data Server support,
```

Comment:
```suggestion
 *		   compiled with Large Composition Data server support,
```

Reply:
Will align this in the other param descriptions as well (light servers) then.

Reply:
Why? 'Server' is capitalized. [See](https://github.com/nrfconnect/sdk-zephyr/blob/38452400d3bed75faf30b5a0e4a5a34d9a44128b/include/zephyr/bluetooth/mesh/health_srv.h#L169)

Reply:
There are about half of them capitalized and half are not. Then up to implementer what to do with this. I do not think this is really important to argue about this.

Reply:
I'll align capitalization where I find it (I.e. leave them capitalized and amend the times it is not). Agree it is not that important, but nice to have it same everywhere.